### PR TITLE
feat: Added help page

### LINKED ISSUE #2004: Add help option in the skills Activity menu
**Describe the feature you'd like**
Help option should be available in the menu bar. This would redirect to help Activity. This will provide guide to the user, whenever they want help.

**Additional context**
It will increase the ease to use the app for the user.

**Screenshots**

help

**Would you like to work on the issue?**
Yes

### PR DESCRIPTION
**Pull request by atm1504**
Fixes #2004 

Changes: 
* Added help fragment
* Added datas in string file
* Added few dimensions in the dimen file
* Added an image

Screenshots for the change: 
help3
help2
help1

### CODE REVIEW

**Comment by iamareebjamal** [comment]
Please remove the image

**Comment by iamareebjamal** [comment]
Place individual questions in cards

**Comment by atm1504** [comment]
##Look after adding cardview
h2
h1

**Comment by iamareebjamal** [comment]
No need for the big help text, otherwise LGTM

**Comment by atm1504** [comment]
> No need for the big help text, otherwise LGTM

Shall I remove that text 'Help'?

**Comment by atm1504** [comment]
@iamareebjamal I have removed the heading 'Help'. You can review it now.

File: app/build.gradle [hunk 1]
```diff
@@ -107,6 +107,7 @@ dependencies {
 
     //Leak Canary
     implementation 'com.android.support.constraint:constraint-layout:1.1.3'
+    implementation 'com.android.support:support-v4:28.0.0'
```
Comment (iamareebjamal) [inline app/build.gradle]: Revert all changes in this file. These are unneeded

File: app/src/main/java/org/fossasia/susi/ai/skills/help/HelpFragment.kt [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    }
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        // Inflate the layout for this fragment
+        val thisActivity = activity
+        if (thisActivity is SkillsActivity) thisActivity.title = getString(R.string.action_help)
```
Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/susi/ai/skills/help/HelpFragment.kt]: Why is this needed?

File: app/src/main/res/layout/fragment_about_us.xml [hunk 3]
```diff
@@ -13,19 +13,19 @@
         <LinearLayout
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
-            android:layout_marginBottom="@dimen/margin_extra_large"
             android:layout_marginTop="@dimen/margin_medium"
+            android:layout_marginBottom="@dimen/margin_extra_large"
```
Comment (iamareebjamal) [inline app/src/main/res/layout/fragment_about_us.xml]: Revert all changes in this file

File: app/src/main/java/org/fossasia/susi/ai/skills/help/HelpFragment.kt (hunk 2 above)
Comment (atm1504) [reply app/src/main/java/org/fossasia/susi/ai/skills/help/HelpFragment.kt]: Sir, this is required to change the title of the page from "SUSI.AI Skills" to Help

File: app/src/main/res/layout/fragment_about_us.xml (hunk 3 above)
Comment (atm1504) [reply app/src/main/res/layout/fragment_about_us.xml]: I will revert all this changes

File: app/src/main/java/org/fossasia/susi/ai/skills/help/HelpFragment.kt (hunk 2 above)
Comment (iamareebjamal) [reply app/src/main/java/org/fossasia/susi/ai/skills/help/HelpFragment.kt]: Why is the cast to Skills Activity needed?

**Comment by atm1504** [comment]
@iamareebjamal sir please review it.

**Comment by iamareebjamal** [comment]
What if I go back to previous fragment, is title changed correctly?

**Comment by atm1504** [comment]
> What if I go back to previous fragment, is title changed correctly?

Yes sir it is changing correctly

**Comment by iamareebjamal** [comment]
@mariobehling The font is Roboto by default. The custom font you're seeing is used by @atm1504, it'll be Roboto by default on every device.

@atm1504 Please revert the changes, the font family is no op on many devices and can crash on MIUI and Samsung phones

**Comment by atm1504** [comment]
@iamareebjamal okk I will revert it

**Comment by atm1504** [comment]
@iamareebjamal please merge it
